packages/codec/lib/compilations/utils.ts

```diff
@@ -22,8 +22,8 @@ export function shimCompilations(
   );
 }
 
-export function shimCompilation(
-  inputCompilation: Common.Compilation,
+function shimCompilation(
```

Comment:
I'd leave the `export`s on these functions; they may not currently be used externally but they were written with the intent that they might be.

Reply:
Fair enough, I'll leave them exported as they were.